lib/agendash.js

```diff
@@ -81,15 +81,23 @@ module.exports = function (agenda, options) {
       /*
       TODO: Need to validate user input.
       */
-      agenda.create(jobName,jobData)
-        .schedule(jobSchedule)
-        .repeatEvery(jobRepeatEvery)
-        .save(function (err) {
-          if (err) {
-            callback('Jobs not created')
-          }
-          callback()
-        });
+      var job = agenda.create(jobName,jobData)
```

Comment:
@HugoCornu You`re right. I was misread the codes.
how about this? it may be meaningless to discuss, but I prefer positive-first to negative-first.

```javascript
if (jobSchedule && jobRepeatEvery) {
  job.repeatAt(jobSchedule)
  job.repeatEvery(jobRepeatEvery)
} else if (jobSchedule) {
  job.schedule(jobSchedule)
} else if (jobRepeatEvery) {
  job.repeatEvery(jobRepeatEvery)
} else {
  return callback('Jobs not created')
}
```